File: beecrowd/1006 - Média 2.go
package main

import "fmt"

func main() {
	var nota1, nota2, nota3 float64

	fmt.Scan(&nota1)
	fmt.Scan(&nota2)
	fmt.Scan(&nota3)

	media := (nota1*2.0 + nota2*3.0 + nota3*5.0) / 10.0

	fmt.Printf("MEDIA = %.1f\n", media)
}

File: beecrowd/1006 - Média 2.kt
import java.util.*

import java.util.Locale

fun average(n1: Double, n2: Double, n3: Double) = (2.0 * n1 + 3.0 * n2 + 5.0 * n3) / 10.0

fun main(args: Array<String>) {

	val nota1 = readLine()!!.toDouble()
    val nota2 = readLine()!!.toDouble()
    val nota3 = readLine()!!.toDouble()

    println("MEDIA = %.1f".format(Locale.US, average(nota1,nota2,nota3)))
}
